jwt-bearer-auth-yesod/README.lhs

```diff
@@ -95,27 +95,34 @@ module Main ( main ) where
 import Prelude
 
 import Control.Lens
-import Control.Monad (when, (<=<))
 import Crypto.JWT
 import Data.Aeson
 import Data.Aeson.KeyMap
-import Network.Wai.Handler.Warp (defaultSettings, runSettings)
-import System.Environment (getArgs)
-import Web.Auth.Bearer.JWT.Cache
 import Web.Auth.Bearer.JWT.Yesod
-import Web.Auth.Bearer.JWT.Yesod.Lens
 import Yesod.Core
 ```
 
+<!--
+```haskell
+import Control.Monad (when, (<=<))
+import Network.Wai.Handler.Warp (defaultSettings, runSettings)
+import System.Environment (getArgs)
+```
+-->
 
 First, your `App` type (or, what Yesod also calls `site`) will need to be able to
-have a way to access the JWKStore itself. This is done by providing a `lens`.
+have a way to access the `ConfiguredStore`. That's your source of public keys with which to verify
+bearer tokens, including any settings required to set it up.
+
+This is done by providing a `lens`.
 
 ```haskell
-data App = App { appJWKCache :: JWKCache }
+data App = App
+  { appJWKCache :: ConfiguredStore JWKCache
+  }
 
-instance HasJWKStore JWKCache App where
-  jwkStoreL = lens appJWKCache $ \app cache -> app {appJWKCache = cache}
+instance HasConfiguredKeyStore JWKCache App where
```

Comment:
A little late, and it doesn't functionally matter, but I would've ordered the type variables the other way here. Aren't you saying _app has jwkcache_? And isn't the order in the actual lens member `Lens' App JWKCache`?

Reply:
Interestingly, yes I ordered the type variables this way *because* I'm saying "app has configured key store of type jwk cache", so the partially applied `HasConfiguredKeyStore JWKCache` is like a unary predicate on app, as if the type class was `HasConfiguredJWKCache` or `HasConfiguredTokenServerUrl` etc